fastlane/Fastfile

```diff
@@ -44,8 +43,13 @@ private_lane :merged_prs_since_last_release do |options|
   }
 
   result = if ensure_git_log_includes_pr
+    current_branch = sh("git branch --show-current")
+    sh("git fetch")
+    sh("git checkout %s" % base_branch_name)
+    recent_commits = sh("git log --format=\"%H\" -100").split("\n")
```

Comment:
I've dropped this down to 100 (rather than 1000) to reduce the output a bit, and because IMO if we're merging more than 100 commits into a beta then we really should be doing more frequent releases...

Reply:
This generally runs within the context of a GitHub Action, which has already checked out the relevant branch/tag in order to produce a build. Unless that has changed recently I think we might be able to avoid introducing:

```
current_branch = sh("git branch --show-current")
sh("git fetch")
sh("git checkout %s" % base_branch_name)
```

Reply:
I believe iOS `promote_to_production` might be the exception to this? It runs on the selected workflow branch, but needs the release notes from a specific tag.

Reply:
Good point. For consistency, I would probably recommend fixing this in the promote-to-production workflow. We'd need to do something like:

```
    steps:
        - name: Checkout
          uses: actions/checkout@v2
          with: 
              ref: ${{ github.event.inputs.tag_to_promote }}
              fetch-depth: 0
```